Paralayout.podspec

```diff
@@ -3,10 +3,10 @@ Pod::Spec.new do |s|
   s.version  = '2.0.0'
   s.license  = 'Apache License, Version 2.0'
   s.summary  = 'Paralayout is a simple set of assistive UI layout utilities. Size and position your UI with pixel-perfect precision. Design will love you!'
-  s.homepage = 'https://github.com/square/Paralayout'
-  s.authors  = 'Square'
-  s.source   = { :git => 'https://github.com/square/Paralayout.git', :tag => s.version }
+  s.homepage = 'https://github.com/NickEntin/Paralayout'
```

Comment:
if we're going to publish a podspec, it'll need a new name.

edit: I see what you're doing below